infra_tooling_account/infra_tooling_account/infra_tooling_alerting_stack.py

```diff
@@ -9,86 +9,122 @@
     aws_iam as iam,
     aws_s3_deployment as s3deploy,
     aws_sqs as sqs,
-    Duration
+    Duration,
 )
 from constructs import Construct
 import os
+import json
 
 
 class InfraToolingAlertingStack(Stack):
-
     def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
-
         stage_name = kwargs.pop("stage_name", None)
         project_name = kwargs.pop("project_name", None)
 
         super().__init__(scope, construct_id, **kwargs)
-        
-        input_timestream_database_arn = Fn.import_value(f"output-{project_name}-metrics-events-storage-arn-{stage_name}")
 
+        input_timestream_database_arn = Fn.import_value(
+            f"output-{project_name}-metrics-events-storage-arn-{stage_name}"
+        )
 
         # Settings S3 bucket
         settings_bucket_name = f"s3-{project_name}-settings-{stage_name}"
         settings_bucket = s3.Bucket(
             self,
             "salmonSettingsBucket",
             bucket_name=settings_bucket_name,
-            block_public_access=s3.BlockPublicAccess.BLOCK_ALL
+            block_public_access=s3.BlockPublicAccess.BLOCK_ALL,
         )
 
         # S3 settings files deployment
         s3deploy.BucketDeployment(
-            self, "salmonSettingsDeployment",
-            sources=[s3deploy.Source.asset('../config/settings')],
+            self,
+            "salmonSettingsDeployment",
+            sources=[s3deploy.Source.asset("../config/settings")],
             destination_bucket=settings_bucket,
-            destination_key_prefix='settings'
+            destination_key_prefix="settings",
         )
 
         # EventBridge Bus
         notification_bus = events.EventBus(
             self,
             "salmonNotificationsBus",
-            event_bus_name=f"eventbus-{project_name}-notification-{stage_name}"
+            event_bus_name=f"eventbus-{project_name}-notification-{stage_name}",
+        )
+
+        # EventBridge bus resource policy
+        # TODO: reuse existing settings reader
+        general_settings_file_path = "../config/settings/general.json"
+        with open(general_settings_file_path) as f:
+            try:
+                general_config = json.load(f)
+            except json.decoder.JSONDecodeError as e:
+                raise json.decoder.JSONDecodeError(
+                    f"Error parsing JSON file {general_settings_file_path}",
+                    e.doc,
+                    e.pos,
+                )
+
+        monitored_account_ids = [
+            account["AccountId"] for account in general_config["monitored_environments"]
```

Comment:
there might be a case when we have 2 monitored env (same account, different regions).
I guess, we might need remove duplicates here

Reply:
Fixed